### ISSUE TITLE
[Manager] Duration default value is not correctly displaying in the Property dialog

### ISSUE THREAD

**Issue by konekoya** [oharastream/ohara#4994]
![註解 2020-05-22 145725](https://user-images.githubusercontent.com/6234750/82641857-9979f200-9c3f-11ea-88ed-86d412fa8f24.png)

DURATION的預設值沒有在畫面上出現

_Originally posted by @chia7712 in https://github.com/oharastream/ohara/issues/4539#issuecomment-632536387_

**Comment by konekoya** [comment]
@chia7712 It looks like the default value for duration is in the correct format (1000 milliseconds) but the response from  POST is returning "PT1S" format.

For example:

`POST` a new perf connector which contains a duration def for "Frequency":

```json
{
  "id": "5d40c99f-4de3-4bf9-b410-809a50f646c5",
  "name": "perf",
  "workerClusterKey": {
    "name": "workspace1",
    "group": "worker"
  },
  "group": "d68857eba",
  "connector.class": "oharastream.ohara.connector.perf.PerfSource"
}
```

The response body:
```json
{
  "header.converter": "org.apache.kafka.connect.converters.ByteArrayConverter",
  "check.rule": "NONE",
  "tags": {},
  "value.converter": "org.apache.kafka.connect.converters.ByteArrayConverter",
  "perf.batch": 10,
  "perf.frequency": "PT1S",
  "id": "5d40c99f-4de3-4bf9-b410-809a50f646c5",
  "aliveNodes": [],
  "columns": [],
  "workerClusterKey": {
    "group": "worker",
    "name": "workspace1"
  },
  "tasksStatus": [],
  "kind": "source",
  "group": "d68857eba",
  "author": "root",
  "topicKeys": [],
  "name": "perf",
  "producer.override.partitioner.class": "oharastream.ohara.kafka.RowDefaultPartitioner",
  "key.converter": "org.apache.kafka.connect.converters.ByteArrayConverter",
  "lastModified": 1590134700047,
  "perf.cell.length": 10,
  "tasks.max": 1,
  "nodeMetrics": {},
  "connector.class": "oharastream.ohara.connector.perf.PerfSource",
  "revision": "cadeee808d907aa1525fcd63c7b2b1728644389f",
  "version": "0.10.0-SNAPSHOT"
}
```

**Comment by chia7712** [comment]
@konekoya see #4998

**Comment by konekoya** [comment]
Fixed in #4998
